README.md

```diff
@@ -73,81 +171,81 @@ network disconnection or other failure.
 command handler method being called. This significantly reduces testing effort and human error.
 
-### Event processing
+## Event processing
 Axon provides two types of event processors,
 [subscribing and tracking](https://docs.axoniq.io/reference-guide/configuring-infrastructure-components/event-processing/event-processors).
 
 Subscribing processors execute on the same thread that is publishing the event. This allows command dispatching to wait
 until the event has been both appended to the event store and all event handling is completed. Commands are queued for
-processing on a FIFO basis. It is important, therefore, to not use them for long running tasks.  
+processing on a FIFO basis. It is important, therefore, to not use them for long running tasks.
 
-Tracking event processors (TEPs), in contrast, execute in their own thread, monitoring the event store for new events. 
-TEPs track their progress consuming events using tracking tokens persisted in the database. TEPs hold ownership of the 
-tokens, preventing multiple application instances from concurrently performing the same processing. Token ownership 
+Tracking event processors (TEPs), in contrast, execute in their own thread, monitoring the event store for new events.
+TEPs track their progress consuming events using tracking tokens persisted in the database. TEPs hold ownership of the
+tokens, preventing multiple application instances from concurrently performing the same processing. Token ownership
 passes to another application instance in the event that a token owner is shutdown or restarted.
 
-TEPs introduce additional complexity by not guaranteeing that projections will be up to date when an API call has ended. 
-TEPs should, in our opinion, be only used for longer running processing, during replays and when preparing projections 
-for new feature releases. 
+TEPs introduce additional complexity by not guaranteeing that projections will be up to date when an API call has ended.
+TEPs should, in our opinion, be only used for longer running processing, during replays and when preparing projections
+for new feature releases.
 
-Axon also introduces the concept of processing groups as a way of segmenting and orchestrating event processing, ensuring 
-that events are handled sequentially within a group. By default, Axon assigns each tracking event processor (TEP) to its 
-own processing group, aiming to parallelise event processing as much as possible. We take a more conservative approach to 
-make the system easier to reason about by defaulting to subscribing event processors and assigning them to a default 
-processing group unless explicitly assigned elsewhere.  
+Axon also introduces the concept of processing groups as a way of segmenting and orchestrating event processing, ensuring
+that events are handled sequentially within a group. By default, Axon assigns each tracking event processor (TEP) to its
```

Comment:
A few comments:
* Segment has a specific meaning in Axon, which is for segment workers within a single processor. 
* IIUC, processing group is really just a way to assign handlers to event processor. After the processor is assigned (at startup time), it does not do anything anymore. 
* Events are not necessarily sequential within a group. It's true for SEP, but not TEP. TEP can have multiple segment workers.
* The default behaviour of "one processing group per event processor" is applicable to SEP as well.

So maybe change it to something like the follows:
Axon also introduces the concept of processing groups and segments (TEP only) as ways to achieve high concurrency in event processing. By default, Axon assigns each event processor to its own processing group, ...